src/components/Wizard/Wizard.test.js

```diff
@@ -42,58 +29,293 @@ test('Wizard loading renders properly', () => {
     </Row>
   );
 
-  const tree = component.toJSON();
-  expect(tree).toMatchSnapshot();
+  expect(component.render()).toMatchSnapshot();
 });
 
-test('Wizard embedded renders properly', () => {
-  const onStepClick = jest.fn();
-  const onSidebarItemClick = jest.fn();
-  const activeStepIndex = 0;
-  const activeSubStepIndex = 0;
-  const onNextButtonClick = jest.fn();
-  const onBackButtonClick = jest.fn();
-
-  const component = renderer.create(
-    <Wizard embedded>
+test('Wizard steps renders properly', () => {
+  let eventCount = 0;
+  const component = mount(
+    <Wizard show>
       <Wizard.Header title="Wizard Title" />
       <Wizard.Body>
         <Wizard.Steps
-          steps={renderWizardSteps(mockWizardItems, 0, 0, onStepClick)}
+          steps={[
+            <Wizard.Step
+              key={0}
+              stepIndex={0}
+              step={0}
+              label="Step 1"
+              title="Step 1"
+              onClick={() => eventCount++}
+            >
+              <Wizard.SubStep key={0} subStep="1.1" title="Step 1.1" />
+            </Wizard.Step>
+          ]}
         />
+      </Wizard.Body>
+    </Wizard>
+  );
+
+  component
+    .find('.wizard-pf-step a')
+    .first()
+    .simulate('click');
+
+  expect(eventCount).toBe(1);
+  expect(component.render()).toMatchSnapshot();
+});
+
+test('Wizard sidebar renders properly', () => {
+  let eventCount = 0;
+
+  const component = mount(
+    <Wizard show>
+      <Wizard.Header title="Wizard Title" />
+      <Wizard.Body>
         <Wizard.Row>
           <Wizard.Sidebar
-            items={renderSidebarItems(
-              mockWizardItems,
-              activeStepIndex,
-              activeSubStepIndex,
-              onSidebarItemClick
-            )}
+            items={[
+              <Wizard.SidebarGroup key="1" step="1" activeStep="1">
+                <Wizard.SidebarGroupItem
+                  className="wizard-sidebar-group-item"
+                  key="1.1"
+                  stepIndex={1}
+                  subStepIndex={1}
+                  subStep="1.1"
+                  label="1A."
+                  title="Details"
+                  activeSubStep="1.1"
+                  onClick={() => eventCount++}
+                />
+              </Wizard.SidebarGroup>
+            ]}
           />
+        </Wizard.Row>
+      </Wizard.Body>
+    </Wizard>
+  );
+
+  component
+    .find('.wizard-sidebar-group-item a')
+    .first()
+    .simulate('click');
+
+  expect(eventCount).toBe(1);
+  expect(component.render()).toMatchSnapshot();
```

Comment:
Same Issue for this snapshot.  Snapshots allow you to pass a string to them for a name.  It is useful if you have multiple snapshots.